3.2/root/usr/share/container-scripts/mongodb/common.sh

```diff
@@ -253,6 +174,26 @@ within the container or visit https://github.com/sclorgk/mongodb-container/."
   exit 1
 }
 
+# process_files process extending files in $1 and $2 directories
+# - source all *.sh files
+#   (if there are files with same name source only file from $1)
+function process_files() {
+  local custom_dir default_dir
+  custom_dir=$1
+  default_dir=$2
+
+  shopt -s nullglob
+  for filename in $(echo $custom_dir/*.sh $default_dir/*.sh | xargs -r basename -a | sort | uniq); do
```

Comment:
I'd rather use `find` command instead of `echo $dir | xargs`... e.g. `find $custom_dir $default_dir -type f -name '*.sh' -printf "%f\n" | sort -u` should do similar thing I hope.